themes/dark/templates/editShow.mako

```diff
@@ -216,4 +220,130 @@
         </form>
     </div>
 </div>
+
+</%block>
+<%block name="scripts">
+    <script type="text/javascript" src="js/quality-chooser.js?${sbPID}"></script>
+    <script type="text/javascript" src="js/edit-show.js"></script>
+<script src="js/lib/vue.js"></script>
+<%include file="/vue-components/selectListUi.mako"/>
+<%include file="/vue-components/anidbReleaseGroupUi.mako"/>
+<script>
+var startVue = function() {
+    const app = new Vue({
+        el: '#config-content',
+        data() {
+            // Python conversions
+            // JS only
+            const exceptions = [];
+            return {
+                seriesSlug: $('#series-slug').attr('value'),
+                seriesId: $('#series-id').attr('value'),
+                indexerName: $('#indexer-name').attr('value'),
+                config: MEDUSA.config,
+                exceptions: exceptions,
+                seriesObj: {
+                    config: {
+                        dvdOrder: false,
+                        flattenFolders: false,
+                        anime: false,
+                        scene: false,
+                        sports: false,
+                        paused: false,
+                        location: "",
+                        airByDate: false,
+                        subtitlesEnabled: false,
+                        release: {
+                            requiredWords: null,
+                            ignoredWords: null,
+                            blacklist: [],
+                            whitelist: []
+                        }
+                    },
+                    language: ''
+                },
+                defaultEpisodeStatusOptions: [
+                    {text: 'Wanted', value: 'Wanted'},
+                    {text: 'Skipped', value: 'Skipped'},
+                    {text: 'Ignored', value: 'Ignored'}
+                ],
+                saveStatus: '',
+                seriesLoaded: false
+            }
+        },
+        async mounted() {
+            const seriesSlug = $('#series-slug').attr('value');
+            const url = 'series/' + seriesSlug;
+            try {
+                const response = await api.get('series/' + this.seriesSlug);
+                this.seriesObj.config = response.data.config;
+            } catch (error) {
+                console.log('Could not get series info for: '+ seriesSlug);
+            }
+        },
+        methods: {
+            anonRedirect: function(e) {
+                e.preventDefault();
+                window.open(MEDUSA.info.anonRedirect + e.target.href, '_blank');
+            },
+            prettyPrintJSON: function(x) {
+                return JSON.stringify(x, undefined, 4)
+            },
+            loadSeries: async function() {
+
+            },
+            saveSeries: async function(subject) {
+                if (['seriesObj', 'all'].includes(subject)) {
+                    const data = {
```

Comment:
Switch this to using deconstructing like I showed in slack.

Reply:
okay, still need to do this one